route/note-route.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const Note = require('../model/note');
+const response = require('../lib/response');
+
+var notePool = {};
+
+module.exports = function(router){
+  router
+  //Create New
+  .post('/api/note', function(req, res){
+    if(req.body){
+      const note = new Note(req.body.content);
+      notePool[note.id] = note;
+      return response(200, note)(res);
+    }
+    response(400, 'bad request')(res);
+  })
+
+  .get('/api/note', function(req, res){
+    const note = notePool[req.url.query.id];
```

Comment:
This is great for lab 07, for lab 08 you'll want to implement your storage module's fetchItem here to get the new note into storage.